package/MDAnalysis/core/groups.py

```diff
@@ -354,16 +354,13 @@ class GroupBase(_MutableBase):
     |                               |            | contain the same elements  |
     |                               |            | in the same order          |
     +-------------------------------+------------+----------------------------+
-    | ``x in s``                    |            | test if ``x`` is part of   |
-    |                               |            | ``s``                      |
-    +-------------------------------+------------+----------------------------+
-    | ``x not in s``                |            | test if ``x`` is not part  |
```

Comment:
Is it worthwhile pointing out in the docs that this works as expected? 

I am fine with leaving it out as you did, given that everything "just works"™ in a pythonic fashion.